setup/azurecmdfiles.wxi

```diff
@@ -1905,6 +1988,9 @@
                         <Component Id="cmp8D637393B06DDB4E95F9C9DE02FBDC88" Guid="*">
                             <File Id="fil2E0A33EC74AA6C08679F32E280E41EC3" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Profile\Microsoft.Threading.Tasks.Extensions.dll" />
                         </Component>
+                        <Component Id="cmpD9121909DC0B824088B2A5EF17CA911C" Guid="*">
```

Comment:
Missed this, please remove these.  This looks like it's caused by a later version of the WebDeploy feature.  There is a PR we are lookign at to remove this, but please remove these in the meantime, we cannot ship Microsoft.Web.Administration, as it is a platform-specific dll